backend/build.gradle.kts

```diff
@@ -1,12 +1,16 @@
+val logbackVersion = "1.2.+"
```

Comment:
Nemam rad to plusko nebo hvezdicku. Nemas to pod kontrolou mezi buildama, bal bych se regresi zpusobene drobnymi nekompatibilitama. Chapu, ze to znamena usporu pri udrzbe a manualniho usili.
@martin-ladecky nazor? :-)

Reply:
Jo, to mi prijde, ze podle semantic versioning je minor coz znamena backward compatible changes, v praci ted to tak name a zatim je to dobre, ale kdyz se ti nezda, muzeme dat pevnou

Reply:
ja vim minor verze by meli byt v pohode. tak muzeme pockat, co si o tom mysli Martin jakozto lead :-)

Reply:
Ahoj,
Martin ako paranoidný Javista preferuje presnú verziu. 
Dôvody:
1. Aj minor verzia môže niečo rozbiť (áno, videl som to viackrát)
2. Ak aj nová verzia nič nerozbije, tak je otázka, čo získame minor aktualizáciou. Prakticky nič. Ak už je problém, tak sa vydávajú nové verzie (viď log4j). To isté platí aj v prípade performance vylepšení.

Za mňa teda preferencia presnej verzie. Build je stále konzistentný. Ak zistíme nejaký problém, môžeme ručne aktualizovať.
@d35h , @rposkocil - stačí to takto? Ak áno, uprav prosím na presné číslo. Ďakujem.

Reply:
Ok, je to tam [2ab82b4](https://github.com/cesko-digital/loono-be/pull/60/commits/2ab82b4ecb51576bdcae5d9e78d629e16001cc15)